test_msgs/CMakeLists.txt

```diff
@@ -21,9 +21,11 @@ find_package(action_msgs REQUIRED)
 find_package(rosidl_default_generators REQUIRED)
 
 set(msg_files
+  "msg/BoundedArrayBoundedArrayPrimitivesNested.msg"
```

Comment:
For consistency, I would name this `BoundedArrayPrimitivesNested`.

Reply:
Makes sense, our suggestion was motivated by `DynamicArrayStaticArrayPrimitivesNested.msg`. But I would prefer your shorter suggestion.